### DISCUSSION TITLE
pi-Base community call Thu 2024 Aug 15 at 3pm Eastern

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#690] [Announcements]
Agenda:

- potential `notation:` and `keywords:` and `abbreviations:` fields https://github.com/pi-base/web/issues/159
- threshold for modifying results from literature without external discussion (e.g. without posting to Math SE): https://github.com/pi-base/data/pull/676
- stating theorems in full generality or not: see https://github.com/pi-base/data/pull/676#issuecomment-2219580551 and later comments
- discuss https://github.com/pi-base/data/pull/658
- locally Euclidean or locally $n$-Euclidean? https://github.com/pi-base/data/pull/706

Zoom link: https://southalabama.zoom.us/j/93833246214

**Comment by ccaruvana** [comment]
I would prefer August 15 since I have a commitment on the first.

**Reply by StevenClontz** [reply]
We'll do the 15th!

**Comment by StevenClontz** [comment]
@prabau consider what I wrote at https://github.com/pi-base/data/wiki/Reviewing#:~:text=We%20like%20for%20theorems%20to%20be%20generalized for the topic of generalizing theorems

**Reply by prabau** [reply]
That looks fine.

One minor thing in the section "Requirements for a new property".  Shouldn't the sentence "If they don't exist, these dependent properties should be ..." be changed to "If they don't exist, these other properties should be ..."?  (because the _dependent_ property would the one that depends on the other ones, namely the new property being introduced, but we mean the ones that the new property depends on instead)

**Reply by StevenClontz** [reply]
done
